apps.json

```diff
@@ -516,12 +546,16 @@
         "background-color": "#f44336"
       }
     },
-    {"name": "Misc",
-     "image": "misc.png",
+    {"name": "About Project",
+     "image": "about.png",
      "style" : {
-        "background-color": "#039BE5"
+        "background-color": "#e91e63"
       }
-    }   
+<<<<<<< HEAD
+    }
+=======
+    }     
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/apps.json
+++ b/apps.json
@@ -554,7 +554,7 @@
 <<<<<<< HEAD
     }
 =======
-    }     
+    }
 >>>>>>> 0f6699c... Fixed html lint errors
   ],
   "category": {
```